.github/actions/install-build-tools/action.yml

```diff
@@ -14,6 +14,7 @@ runs:
       export BUILD_TOOLS_SHA=8246e57791b0af4ae5975eb96f09855f9269b1cd
       npm i -g @electron/build-tools
       e auto-update disable
+      e d auto-update disable
```

Comment:
What's the difference between these two lines, can we nix one of them?

Reply:
One disables build tools updates; the other disables depot_tools updates.

Reply:
https://github.com/electron/build-tools/blob/main/src/e-depot-tools.js#L24 - disables depot_tools updates

https://github.com/electron/build-tools/blob/7379782a48976ba64172fe17b931909631adaed3/src/e-auto-update.js#L17 - disables build tools updates